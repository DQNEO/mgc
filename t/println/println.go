package main

func f1() {
	println("0")
	println("1")
	println("256")
}

func f2() {
	println(0)
	println(1)
	println(256)
}

func main() {
	f1()
	f2()
}
