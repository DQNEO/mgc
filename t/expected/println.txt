0
1
256
0
1
256
